src/backend/distributed/test/distributed_intermediate_results.c

```diff
@@ -118,7 +118,7 @@ redistribute_task_list_results(PG_FUNCTION_ARGS)
 	bool binaryFormat = PG_GETARG_BOOL(3);
 
 	Query *parsedQuery = ParseQueryString(queryString, NULL, 0);
-	PlannedStmt *queryPlan = pg_plan_query_compat(parsedQuery, queryString,
```

Comment:
Replacing `queryString` with `NULL` seems incorrect

Reply:
why?

Reply:
The function is supposed to receive the queryString in pg13. It eventually calls planner_hook. Other future extensions might rely on receiving a non-NULL queryString

Reply:
There were also other calls where we are sending queryString -> NULL.

I am reverting the change now.